nvtabular/dask/io.py

```diff
@@ -228,6 +231,9 @@ def __init__(self, paths, part_size, fs, fs_token):
     def to_ddf(self, columns=None):
         raise NotImplementedError(""" Return a dask_cudf.DataFrame """)
 
+    def to_iter(self, columns=None):
```

Comment:
This method is 'to_itr' in all the engine subclasses and 'to_iter' here and in the dataset class on line 214. Can we be consistent on how we name this? I'd prefer to_iter fwiw